src/components/Tag/index.tsx

```diff
@@ -19,6 +19,19 @@ const DeleteButton = styled(Button)`
 	color: ${(props) => props.theme.colors.tertiary.semilight};
 `;
 
+const tagItemsToString = (items: TagItem[]) => {
+	return items.reduce((itemsString, item, index) => {
```

Comment:
```suggestion
	return items.map((item, index) => {
	       const prefix = index > 0 ? `${item.prefix || ','} ` : '';
         return prefix + item.value ? `${item.name} ${item.operator ?? ':'} ${item.value}` : item.name)
     }
	 .join('\n')
```

Reply:
I think something like this would work and is slightly more concise (and cleaner than using a reducer)

Reply:
Ah - yep, I like that much better!